Dockerfile

```diff
@@ -5,3 +5,4 @@ WORKDIR /agora-data-tools
 COPY . .
 
 RUN pip install --no-cache-dir .
+RUN apt-get install -y procps
```

Comment:
Thanks @BWMac . I would move this up to line 2 and do something like this:
```
RUN  apt-get update && \
  apt-get install -y procps && \
  rm -rf /var/lib/apt/lists/*
```

Reply:
Done in latest commit